commands/token.js

```diff
@@ -0,0 +1,27 @@
+var exec = require('child_process').exec;
+var open = require('open');
+
+module.exports = function () {
+  exec('git remote show origin', { cwd: __dirname }, function (err, output) {
```

Comment:
Scores 96 with no vulnerabilities so I'm good with it :-).
https://www.bithound.io/github/wilmoore/node-git-origin-url